components/test_raftstore/src/pd.rs

```diff
@@ -716,7 +716,7 @@ impl TestPdClient {
 
     pub fn must_have_peer(&self, region_id: u64, peer: metapb::Peer) {
         for _ in 1..500 {
-            sleep_ms(10);
+            sleep_ms(50);
```

Comment:
Which test does this PR try to stable/fix?

Reply:
see #3902